pkg/app/args.go

```diff
@@ -31,13 +30,8 @@ func CreateArgs() (Args, error) {
 func validateArgs(args *Args) error {
 	var err error
 	if args.ConfigFile == "" {
```

Comment:
Yup this never gets used. Turns out we never even get to this code if there's any error in the args.